src/Engine/ProtoCore/Lang/Type.cs

```diff
@@ -81,7 +81,8 @@ public override string ToString()
         /// <returns></returns>
         public string ToShortString()
         {
-            if (!string.IsNullOrEmpty(Name) && Name.Contains("."))
+
+            if (!string.IsNullOrEmpty(Name) && Name.Contains(".") && !Name.Contains("System"))
```

Comment:
@mjkkirschner It's the `System` type that fails in the custom node workspace. `Autodesk` namespace works fine. 
I am not sure whether this is the right fix. Otherwise, the fix is to use  `ToString` and fix the tooltip to show `point` instead of `Autodesk.Geometry.point`.

Reply:
I don't think this is correct, - this is too specific and doesn't address the underlying problem, there are other namespaces besides system that people will put in their custom node inputs.... 

I think it's fine if the tooltip shows the full type name or if we add a new property or something.